spec/models/gids_redis_spec.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'common/exceptions'
+
+describe GIDSRedis do
+  subject { GIDSRedis.new }
+
+  let(:scrubbed_params) { {} }
+  let(:body) { {} }
+  let(:gids_response) do
+    GI::GIDSResponse.new(status: 200, body: body)
+  end
+
+  describe 'method_missing' do
+    it 'returns response body' do
+      allow_any_instance_of(GI::Client).to receive(:get_institution_details).and_return(gids_response)
+
+      expect(subject.get_institution_details(scrubbed_params)).to eq(gids_response.body)
+    end
+
+    it 'undefined method' do
+      # expect().to raise_error does not work
+      # expect(subject.not_a_real_method()).to raise_error(NoMethodError)
+      begin
+        subject.not_a_real_method
+      rescue NoMethodError => e
+        expect(e).to be_instance_of(NoMethodError)
+      end
```

Comment:
Try using curly braces instead of parenthesis:
```ruby
expect{ subject.not_a_real_method }.to raise_error(NoMethodError)
```

Reply:
changed, thanks